### DISCUSSION TITLE
nearbyhotel button

### DISCUSSION THREAD

**Discussion by ramprasath2102** [Imperial-lord/mapbox-flutter#7] [General]
hello bro, in the first video the list of hotels with polylines....in this case if i opened a app it automatically shows me carousel and polyline but i want to put a button like nearby hotle if i clicked the button the app want to show carosel and polyline

**Comment by Imperial-lord** [comment]
Instead of adding those in the onStyleLoadedCallback(), you need to put them in a separate function, and then call the function inside the onPressed of your button.
